buildSrc/build.gradle.kts

```diff
@@ -72,10 +72,6 @@ gradlePlugin {
             id = "LicenseResolverPlugin"
             implementationClass = "com.google.firebase.gradle.plugins.license.LicenseResolverPlugin"
         }
-        register("dackkaPlugin") {
-            id = "DackkaPlugin"
-            implementationClass = "com.google.firebase.gradle.plugins.DackkaPlugin"
-        }
```

Comment:
Hm, how come this plugin is being removed?

Reply:
I mentioned it in the pr description, but basically it serves no purpose. I didn't realize you don't need to actually register plugins to use them. This way, people don't accidentally apply the plugin- and potentially cause issues down the line cause of a misunderstanding on how the plugin works. We can always add it back if need be fwiw